torchao/float8/float8_linear.py

```diff
@@ -191,15 +188,6 @@ def __init__(self, *args, **kwargs):
         # would be initialized in every iteration.
         self.enable_pre_and_post_forward = self.config.enable_pre_and_post_forward
 
-        # See the comments in config.py for more details of this option.
```

Comment:
technically not related to this PR, but making the test logs non-spammy for now and we can add this back in a better way later